hack/update-build.sh

```diff
@@ -0,0 +1,74 @@
+
+#!/usr/bin/env bash
+
+echo "Updating container file"
+
+: "${COMMIT:=$(git rev-list --abbrev-commit --tags --max-count=1)}"
+: "${CONTAINER_FILE:=./Dockerfile}"
+: "${BUNDLE_CONTAINER_FILE:=./bundle.Dockerfile}"
+: "${CATALOG_CONTAINER_FILE:=./catalog.Dockerfile}"
+: "${TARGET_VERSION:=1.7.0}"
+: "${REPLACE_VERSION:=1.6.0}"
```

Comment:
Current latest is 1.6.1

```suggestion
: "${REPLACE_VERSION:=1.6.1}"
```